catlin/.gitignore

```diff
@@ -0,0 +1 @@
+catlin
```

Comment:
any reason to ignore `catlin` here :thinking:

Reply:
to make sure we don't commit a binary we compiled while developing on it I think 😉

Reply:
that's correct :-p 

I actually have another followup pr that adds a proper makefile with linting and generation etc..

Reply:
:+1: